src/views/PageSearchPortal.vue

```diff
@@ -85,11 +91,22 @@ export default {
         ]),
         async getParkingRequests() {
             this.isLoading = true;
-            const parkingRequestList = await mayaClient.get(
-                '/internal/parking-requests',
-            );
-            this.parkingRequests = parkingRequestList;
-            this.isLoading = false;
+            const mobile = this.VOMobile;
+            if (mobile === mobile) {
+                const mobileNo = await mayaClient.get(
+                    `internal/parking-requests?mobile=${mobile}`,
+                );
+                this.parkingRequests = mobileNo;
+                this.isLoading = false;
+            }
+
+            if (mobile === '') {
+                const mobileNo = await mayaClient.get(
+                    '/internal/parking-requests',
+                );
+                this.parkingRequests = mobileNo,
```

Comment:
since this line is common in both the if block, we can restructure the code to move it outside the if block.